src/Server/Auth.fs

```diff
@@ -18,20 +16,26 @@ let login (ctx: HttpContext) = async {
         let user : ServerTypes.UserRights = { UserName = login.UserName }
         let token = JsonWebToken.encode user
 
-        return! Successful.OK token ctx
+        return! Successful.OK token next ctx
```

Comment:
Ok one issue is that this is not working. It's not sending out the text but some weird stream with couple of the and f in it. @gerardtoconnor @alfonsogcnunez any ideas? Obvious fix is to put it in a record and use fablejson.serialize. But it's an issue that many will have coming from suave.

Reply:
It's also the issue why the pr is still red

Reply:
We've the same code and it's working. I think the problem is [in these lines](https://github.com/SAFE-Stack/SAFE-BookStore/blob/804b10f45ce5566b88f7e241a9ab8e7f4ed2a06e/src/Client/pages/Login.fs#L54-L57), we actually changed them to:

```fsharp
        let! data = response.json()
        return data :?> string
```

Makes sense because the client is asking for a JSON response and the server negotiates (not like Suave that directly sent a text response), so the string comes quoted.